packages/web3-connect/src/components/Web3ConnectAccountSelect.utils.ts

```diff
@@ -0,0 +1,49 @@
+import { arraySearch } from "@galacticcouncil/utils"
+import { filter, pipe, sortBy } from "remeda"
+
+import {
+  Account,
+  PROVIDERS_BY_WALLET_MODE,
+  WalletMode,
+} from "@/hooks/useWeb3Connect"
+
+export const isAccountSelected = (
+  currentAccount: Account | null,
+  account?: Account | null,
+) => {
+  if (!currentAccount || !account) return false
+  return (
+    currentAccount?.address === account.address &&
```

Comment:
can be replaced by `currentAccount.address`, since `currentAccount` cannot be null. The same for provider